crates/biome_cli/src/execute/traverse.rs

```diff
@@ -691,6 +700,11 @@ fn handle_file(ctx: &TraversalOptions, path: &BiomePath) {
         Ok(Ok(FileStatus::Unchanged)) => {
             ctx.increment_unchanged();
         }
+        Ok(Ok(FileStatus::Search(msg))) => {
+            ctx.increment_changed(path);
```

Comment:
What purpose is there for bumping `changed`? It feels semantically off, and I don't see why it's necessary...

Reply:
My bad, I meant to increment the unchanged files, aka when a match is found in a file it will still report that file as "searched"